src/DateTimePicker.jsx

```diff
@@ -175,8 +175,8 @@ var DateTimePicker = React.createClass({
         ref="element"
         tabIndex={'-1'}
         onKeyDown={tetherPopup ? null : this._keyDown}
-        onFocus={tetherPopup ? () => this.setState({focused: true}) : () => this._focus.bind(null, true)}
-        onBlur={tetherPopup ? () => this.setState({focused: false}) : () => this._focus.bind(null, false)}
+        onFocus={tetherPopup ? () => this.setState({focused: true}) : this._focus.bind(null, true)}
+        onBlur={tetherPopup ? () => this.setState({focused: false}) : this._focus.bind(null, false)}
```

Comment:
Reverts `else` condition to exact previous state: https://github.com/procore/react-widgets/blob/3be1b7dd5835c4d54bb8366bbb23ccaf181a90ca/src/Combobox.jsx#L176